velox/type/Timestamp.cpp

```diff
@@ -255,6 +255,8 @@ std::string Timestamp::tmToString(
     const TimestampToStringOptions& options) {
   VELOX_DCHECK_GE(nanos, 0);
   VELOX_DCHECK_LT(nanos, 1'000'000'000);
+  double kNanosecondsInMillisecond = 1'000'000;
```

Comment:
Why are these of type double? nanos is of type uint64_t and the division is by a constant (which is actually already defined in Timestamp.h ` static constexpr int64_t kNanosecondsInMillisecond = 1'000'000;`) So they should be used instead of the literals.

Reply:
The purpose is for accurate rounding up. The existing integer division would truncate the decimal, I want to preserve the decimal and then round to the nearest integer for a more accurate operation.